### ISSUE TITLE
Select Element defaults to string "<none>" when selecting <none> as default

### ISSUE THREAD

**Issue by ManuelDittmar** [bpmn-io/form-js#562]
### Describe the Bug

it is possible to select default value <none> which will result in the variable value "<none>". 

### Steps to Reproduce

1. Input:
`{}`

2. Drag and Drop Select Element

![image](https://user-images.githubusercontent.com/83572388/224745039-5a9a82c3-8c8f-4b4b-bd47-20d478aa98d8.png)

3. Observe Form Output

```
{
  "field_0mzhnhz": "",
  "field_16exx0r": null
}
```

4. Set Default to <none>

![image](https://user-images.githubusercontent.com/83572388/224744557-bfd602eb-554c-49b6-9912-edd403766169.png)

5. Resulting Schema:
```
{
  "components": [
    {
      "values": [
        {
          "label": "First",
          "value": "Value"
        }
      ],
      "label": "Select",
      "type": "select",
      "id": "Field_0jqm2fz",
      "key": "select",
      "defaultValue": "<none>"
    },
    {
      "label": "Text field",
      "type": "textfield",
      "id": "Field_1igdsio",
      "key": "text",
      "appearance": {}
    }
  ],
  "schemaVersion": 7,
  "exporter": {
    "name": "Camunda Web Modeler",
    "version": "06b1b8a"
  },
  "type": "default",
  "id": "DefaultValues",
  "executionPlatform": "Camunda Cloud",
  "executionPlatformVersion": "8.1.0"
}
```
6. See Form Output
![image](https://user-images.githubusercontent.com/83572388/224744557-bfd602eb-554c-49b6-9912-edd403766169.png)

```
{
  "field_0mzhnhz": "",
  "field_16exx0r": "<none>"
}
```

### Expected Behavior

I expect to result in null or an empty string.

### Environment

- Host (Browser/Node version), if applicable: Chrome Version 110.0.5481.180
- OS: [e.g. Windows 10]
- Library version: Via Web Modeler

**Comment by pinussilvestrus** [comment]
We had a similar case in the validation type (https://github.com/bpmn-io/form-js/pull/537) where it only popped up in the products. I'll have a look

**Comment by pinussilvestrus** [comment]
A fix got released via `0.12.2` and will be available once integrated into the products (most likely in the Camunda 8.2.0 release).

/cc @marcosgvieira
